icu4j/main/tests/core/src/com/ibm/icu/dev/test/message2/Args.java

```diff
@@ -0,0 +1,193 @@
+// © 2022 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+
+package com.ibm.icu.dev.test.message2;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import com.ibm.icu.message2.MessageFormatter;
+
+/**
+ * Convenience class that provides the same functionality as
+ * <code>Map.of</code> introduced in JDK 11, which can't be used yet for ICU4J.
+ *
+ * <p>The returned Map is immutable, to prove that the {@link MessageFormatter}
+ * does not change it</p>
+ */
+@SuppressWarnings("javadoc")
+public class Args {
+
+    public static final Map<String, Object> NONE = new HashMap<>();
+
+    public static Map<String, Object> of(
+            String argName0, Object argValue0
+    ) {
```

Comment:
nit/optional: I would pull these `) {` to the end of the previous line (multiple instances)

Reply:
Done